src/check-reserve-update-frequency/check-reserve-update-frequency.js

```diff
@@ -10,18 +10,24 @@ const {
   ProtocolDataProvider: protocolDataProviderAddress,
 } = require('../../contract-addresses.json');
 const { abi: protocolDataProviderAbi } = require('../../interfaces/AaveProtocolDataProvider.json');
-const { abi: priceOracleAbi } = require('../../interfaces/IPriceOracle.json');
+const { abi: aaveOracleAbi } = require('../../interfaces/AaveOracle.json');
 const { abi: chainlinkAggregatorAbi } = require('../../interfaces/IChainlinkAggregator.json');
 const { abi: lendingPoolAddressesProviderAbi } = require('../../interfaces/ILendingPoolAddressesProvider.json');
 
-// set up the an ethers provider
-const jsonRpcProvider = new ethers.providers.WebSocketProvider(getJsonRpcUrl());
-
 // time threshold over which we trigger alerts (24 hours = 86400 seconds)
 // this value comes from the Chainlink web interface for price feeds (mouseover Trigger parameters)
 //  'A new trusted answer is written when the off-chain price moves more than the deviation
 //   threshold or 86400 seconds have passed since the last answer was written on-chain.'
-const ORACLE_AGE_THRESHOLD_SECONDS = 86400;
+const {
+  oracleAgeThresholdSeconds: ORACLE_AGE_THRESHOLD_SECONDS,
```

Comment:
this is looking for a key inside another key, the agent-config does not expose the `oracleAgeThresholdSeconds` key directly

Reply:
Import fixed.